docs/diagrams/ModuleViewSequenceDiagram.puml

```diff
@@ -0,0 +1,64 @@
+@startuml
+!include style.puml
+
+actor user
+
+box Ui UI_COLOR_T1
+participant ":MainWindow" as MainWindow UI_COLOR
+end box
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":ViewCommand" as ViewCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+user -> MainWindow : "view cs2103t"
+activate MainWindow
+
+MainWindow -> LogicManager : executeCommand("view cs2103t")
+activate LogicManager
+
+LogicManager -> ViewCommand ** : create and execute
+activate ViewCommand
+
```

Comment:
It might be better to include the ViewCommandParser so it would be easier to compare with the how the rest of the commands work.

Reply:
I disagree, the process is unimportant

Reply:
Sequence diagram show capture how objects interact, in this case ViewCommandParser literally creates ViewCommand, its not exactly a supporting class that supports the creation of ViewCommand. Besides, it is rather misleading at first glance as I initially assumed ViewCommand is created without going through a parser. But if you strongly feel it is sufficient to highlight the process then its OK.

Reply:
The process is abstracted out as it doesn't add much value if left in. End of the day, the creation of the view command is important.